src/components/Character/styles.scss

```diff
@@ -1,19 +1,184 @@
+// Mobile portrait
 .wrapper {
-	font-size: 1em;
+	display: flex;
+	width: 100%;
+	min-height: 100%;
+	padding: 10%; 		//GUESS
+	position: absolute;
+	top: 0;
+	left: 0;
+	flex-direction: column;
 
 	&[aria-hidden="true"]{
 		opacity: 0;
+		pointer-events: none
 	}
 }
-	.button {
-		&.next {
-			font-size: 1em;
+
+	.container {
+		position: relative;
+		z-index: 1;
+	}
+
+		.content {
+			margin-top: 40vh;
+			max-width: 35em;
 		}
-		&.previous {
-			font-size: 1em;
+	
+			.name {
+				margin-bottom: 0.1em; // GUESS
+				font-size: 5em;
+				font-family: var(--font_family__heading);
+			}
+
+			.nickname {
+				margin-bottom: 0.5em; // GUESS
+				font-size: 2.6em;
+				font-family: var(--font_family__heading);
+			}
+
+			.actor {
+				margin-bottom: 1em; // GUESS
+				font-size: 1em;
+				font-family: var(--font_family__body);
+				font-weight: normal;
+				letter-spacing: 0.25em;
+				text-transform: uppercase;
+			}
+
+			.body {
+				margin-bottom: 4em; // GUESS
+				line-height: 1.5;
+				font-family: var(--font_family__body);
+
+				.paragraph+.paragraph {
+					margin-top: 1em;
+				}
+			}
+
+		.content::after {
+			content: "";
+			display: block;
+			width: 100%;
+			height: 12em;
+			position: fixed;
+			bottom: 0;
+			left: 0;
+			background-image: linear-gradient(
+			    rgba(237, 20, 61, 0) 0%,
+			    rgba(237, 20, 61, 1) 50%,
+			    rgba(237, 20, 61, 1) 100%	
+			);
 		}
+
+		.controls {
+			display: flex;
+			width: 100%;
+			max-width: 25rem; // GUESS
+			position: sticky;
+			bottom: 10vw; // GUESS!
+			justify-content: space-between;
+			font-size: 1.1em;
+			font-family: var(--font_family__heading);
+		}
+
+			.button {
+				display: flex;
+				align-items: center;
+				text-decoration: none;
+				--button_margin: 1em;
+
+				&::before, 
+				&::after {
+					display: inline-block;
```

Comment:
Can't do this as it'll show before + after for BOTH prev and next.
i.e. there'll be 4 lovely lime green boxes, instead of two.
I'll do the best i can

Reply:
&.prev::before,
&.next::after {
	content: "";
}

😏

Reply:
ahhh, good shout, and good solution.